include/libdnf/comps/comps.hpp

```diff
@@ -27,19 +27,29 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
 #include "group/group.hpp"
 #include "group/sack.hpp"
 
+extern "C" {
+#include <solv/pool.h>
+#include <solv/repo.h>
+}
```

Comment:
The libsolv includes shouldn't be in a public header.
We're trying to hide libsolv internals from the public API.